src/BlogTitles.jsx

```diff
@@ -1,12 +1,31 @@
-export default function BlogTitles({ titles, onRemove }) {
+import Button from './Button.jsx';
+import styled from 'styled-components';
+
+function BlogTitles({ titles, onRemove, className }) {
   return (
-    <ul>
+    <ul className={className}>
```

Comment:
styled components requires passing the `className` prop to the element you want the style to apply to. Typically we just apply `className` to the root element of the component.